source/modules/MEI.shared.xml

```diff
@@ -7282,7 +7283,9 @@
     </attList>
     <remarks xml:lang="en">
       <p>When an entire element should be rendered in a special way, a style sheet function should
-        be used instead of the <gi scheme="MEI">rend</gi> element.</p>
+        be used instead of the <gi scheme="MEI">rend</gi> element. The <att>glyph.auth</att> and <att>glyph.uri</att>
```

Comment:
Could it be helpful to add this last addition about the use of glyph.auth and glyph.uri to the textual guidelines as well together with an example of how to use it, e.g., with SMuFL?